amundsen_application/static/css/_layouts.scss

```diff
@@ -154,7 +156,7 @@ $resource-header-height: 84px;
   // Inner column layout
   .column-layout-2 {
     display: flex;
-    flex-basis: 100%;
+    padding-top: $aside-separation-space;
```

Comment:
Any reason why the flex-basis was removed? I don't remember when or why it was added thought.
Also the comment above about the padding causing too much space at the top.

Reply:
That was the rule that was adding too much space and triggering a scroll.

Reply:
I see, no problem them. I did some due diligence to try to figure out from the history if `flex-basis` was added for a reason (like to fix a bug) and it doesn't look like it.  May have just been a style we didn't need that was left over from the iterations we went through when we did the resource page redesign last year.